src/java/org/apache/cassandra/db/Keyspace.java

```diff
@@ -243,6 +243,11 @@ public void snapshot(String snapshotName, String columnFamilyName, boolean skipF
             }
         }
 
+        if (SchemaConstants.SYSTEM_KEYSPACE_NAME.equals(getName()) &&
+            columnFamilyName == null ||
+            NodeConstants.canBeMapped(getName(), columnFamilyName))
```

Comment:
According to [Cassandra code guideline](https://cassandra.apache.org/_/development/code_style.html) operators carry to next line for multiline, thus:
```suggestion
        if (SchemaConstants.SYSTEM_KEYSPACE_NAME.equals(getName())
            && columnFamilyName == null
            || NodeConstants.canBeMapped(getName(), columnFamilyName))
```